tests/rkt_exec_test.go

```diff
@@ -43,17 +45,70 @@ func TestRunOverrideExec(t *testing.T) {
 			expectedLine: "inspect execed as: /inspect-link",
 		},
 	} {
-		child, err := gexpect.Spawn(tt.rktCmd)
-		if err != nil {
-			t.Fatalf("%d: cannot exec rkt: %v", i, err)
-		}
+		runRktAndChkOutput(t, tt.rktCmd, tt.expectedLine)
+	}
+}
+
+func TestRunPreparedExec(t *testing.T) {
```

Comment:
please change this to `TestRunPreparedOverrideExec` (since tests are not file-scoped when go test shows their output)